src/components/Dropdown/Dropdown.vue

```diff
@@ -30,34 +30,57 @@
         :aria-required="required"
         :aria-disabled="disabled"
         :class="[
-          'cursor-default bg-white border rounded border-gray-100 focus:outline-none focus:ring-2 focus:ring-primary-100 focus:border-transparent hover:shadow',
+          'cursor-default bg-white border rounded-lg border-gray-100 focus:outline-none focus:shadow hover:shadow font-light text-gray-900',
           {'text-sm py-2 px-2.5': small},
-          {'py-2.5 px-4': default_},
-          {'!bg-white-300 pointer-events-none': disabled},
-          {'border-error': error}
+          {'h-12 py-2.5 px-4': default_},
+          {'!border-error': error},
+          {'!bg-white-100 pointer-events-none': disabled},
+          {'focus:ring-4 focus:ring-primary-100 focus:border-transparent': !open},
+          {'focus:ring-0': open},
+          {'border-gray-500' : open || activeIndex > -1}
         ]"
         tabindex="0"
         @blur="onSelectBlur"
         @click="updateMenuState(!open)"
         @keydown="onSelectKeydown"
       >
-        <span :class="['mr-8', {'text-sm': small}]">
+        <span
+          :class="[
+            'mr-8',
+            {'text-sm': small},
+            {'text-gray-100': disabled},
+            {'text-gray-900' : open || activeIndex > -1},
+            {'text-gray-500' : activeIndex < 0}
+          ]"
+        >
           {{ value || placeholder }}
         </span>
         <chevron-down
+          v-if="!open"
+          :class="[
+            'w-4 h-4 absolute right-2 text-gray-100',
+            {'top-3': small},
+            {'top-4': default_}
+          ]"
+          data-testid="chevron-down"
+        />
+        <chevron-up
+          v-else
           :class="[
-            'w-4 h-4 absolute right-2',
+            'w-4 h-4 absolute right-2 text-gray-100',
             {'top-3': small},
             {'top-4': default_}
           ]"
+          data-testid="chevron-up"
         />
       </div>
       <div
         :id="`${id}-listbox`"
         ref="listbox"
         role="listbox"
         :class="[
-          `bg-white rounded-sm text-sm py-4 overflow-y-auto absolute left-0 top-full hidden w-full z-50 shadow ${listHeight ? `h-${listHeight}` : 'max-h-80'}`,
+          'bg-white rounded-lg text-sm py-4 overflow-y-auto absolute left-0 top-full hidden w-full z-50 shadow h-80',
+          {'custom-list-height': listHeight},
```

Comment:
When we were doing `h-${listHeight}` for some reason any value greater than 12 was being removed by Tailwind.  Adding the `custom-list-height` and using `v-bind` with the `listHeight` variable in the `<style>` block below resolves that issue.